labelbox/pydantic_compat.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+
+def pydantic_import(class_name, sub_module_path: Optional[str] = None):
+    import importlib
+    import pkg_resources
+
+    # Get the version of pydantic
+    pydantic_version = pkg_resources.get_distribution("pydantic").version
+
+    # Check if the version is 1
+    if pydantic_version.startswith("1"):
+        pydantic_v1_module_name = "pydantic" if sub_module_path is None else f"pydantic.{sub_module_path}"
+    else:  # use pydantic 2 v1 thunk
+        pydantic_v1_module_name = "pydantic.v1" if sub_module_path is None else f"pydantic.{sub_module_path}"
```

Comment:
I would assume we want `pydantic.v1` in both cases.  If so, why not?

Reply:
Well what if users are still using pydantic v1? Currently we need to support this case so we will do `from pydantic` instead of `from pydantic.v1`

Reply:
I thought that case is captured by the `if` condition?

I'm referring to the `f"pydantic.{sub_module_path}"` if the `else` condition which assumes the usage of Pydantic 2.x.

Reply:
Good point